internal/cmd/cli/apply.go

```diff
@@ -46,6 +46,7 @@ type Apply struct {
 	SSHPrivateKeyFile           string            `usage:"Path of ssh-private-key for helm repo" name:"ssh-privatekey-file"`
 	HelmRepoURLRegex            string            `usage:"Helm credentials will be used if the helm repo matches this regex. Credentials will always be used if this is empty or not provided" name:"helm-repo-url-regex"`
 	KeepResources               bool              `usage:"Keep resources created after the GitRepo or Bundle is deleted" name:"keep-resources"`
+	DeleteNamespace             bool              `usage:"Delete namespace created after the GitRepo or Bundle is deleted" name:"delete-namespace"`
```

Comment:
I find the usage description a bit confusing here, as if the namespace were created after deletion of the `GitRepo` or bundle. How about something like this?
```suggestion
	DeleteNamespace             bool              `usage:"Delete GitRepo target namespace after the GitRepo or Bundle is deleted" name:"delete-namespace"`
```